src/test/java/com/tstepnik/planner/service/TaskServiceTest.java

```diff
@@ -0,0 +1,32 @@
+package com.tstepnik.planner.service;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TaskServiceTest {
```

Comment:
This made it to the PR. Remove it and remember to test new things/add new things on new branches. You are not tied to one branch at the time.